lib/govuk_app_config/error.rb

```diff
@@ -0,0 +1,15 @@
+require "sentry-raven"
+require "govuk_app_config/statsd"
+
+module GOVUK
+  module Error
```

Comment:
There aren't any conflicts, but on second thought, this is a inconsistent with what other gems provide. Perhaps we're better off with exposing `GovukError` & `GovukStatsd`.